src-electron/db/query-attribute.js

```diff
@@ -165,26 +156,71 @@ async function selectAllAttributeDetailsFromEnabledClusters(
     ATTRIBUTE.NAME,
     ATTRIBUTE.CODE,
     ATTRIBUTE.SIDE,
-    ATTRIBUTE.TYPE,
+    CASE
+       WHEN
+         ATTRIBUTE.ARRAY_TYPE IS NULL
+       THEN
+         ATTRIBUTE.TYPE
+       ELSE
+         ATTRIBUTE.ARRAY_TYPE
+     END AS TYPE,
```

Comment:
But doesn't the mapping handle this now?  Why do we need to handle this in the query _and_ the mapping?

Reply:
removed